app/services/merge/merge_organisations_service.rb

```diff
@@ -87,18 +88,21 @@ def merge_lettings_logs(merging_organisation)
         lettings_log.location = location_to_set if location_to_set.present?
       end
       lettings_log.owning_organisation = @absorbing_organisation
-      lettings_log.save!(validate: false)
+      lettings_log.skip_dpo_validation = true
```

Comment:
I feel like I'm missing something obvious, but does this (/should this) get cleared at any point? (i.e. so that any future saves would not skip this validation)

Reply:
This is just in memory tho, so it should disappear as soon as this lettings_log no longer exists. Or am I missing something?
<img width="441" alt="image" src="https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/assets/54268893/68255a59-e646-449f-a395-b6ac0a6f04cb">